src/configParser.ts

```diff
@@ -135,8 +135,10 @@ export class ConfigParser {
         cktName: cktName,
         inputFilePath: this._getCircuitInputPath(inputDir, c),
         outputDir: cktOutputDir,
+        isTauFileGiven: c.powerOfTauFp? true: false,
         powerOfTauFp: path.resolve(
-          c.powerOfTauFp ? c.powerOfTauFp : path.join(inputDir, `power_of_tau.ptau`) // Calculate total constraints from r1cs file, and download it
+          c.powerOfTauFp ? c.powerOfTauFp : path.join(cktOutputDir, `power_of_tau.ptau`)
```

Comment:
if we keep one file name then it will overwrite the tau file next time. We should name the tau file accordingly.

Reply:
The file path is calculated in `downloadPowerOfTauFile` function. I've updated this code in the latest commit and added a comment here.